tb_pulumi/ec2.py

```diff
@@ -88,6 +87,10 @@ def __init__(
         tags: dict = {},
         **kwargs,
     ):
+        if 'exclude_from_project' in kwargs:
+            exclude_from_project = kwargs['exclude_from_project'] or False
```

Comment:
```suggestion
            exclude_from_project = kwargs.pop('exclude_from_project', False)
```

pop is handy! Just make sure to always include a default, otherwise it will raise an exception if the key doesnt exist. (boo)

Reply:
oh yeah, with this you can remove the del line